Scala/Hangman.scala

```diff
@@ -0,0 +1,294 @@
+import java.sql.{Connection, DriverManager}
+
+/**
+  * Created by Administrator on 26/06/2017.
+  */
+object Main
+{
+  def main(args: Array[String]): Unit = {
+    DBUpdate("CREATE TABLE IF NOT EXISTS wordTable(theID INT(10) PRIMARY KEY AUTO_INCREMENT, theWord VARCHAR(20));")
+
+    DBUpdate("TRUNCATE TABLE wordTable")
+
+    DBUpdate("ALTER TABLE wordTable AUTO_INCREMENT=0")
+
+    DBUpdate("LOAD DATA LOCAL INFILE 'src/main/resources/words_alpha.txt' INTO TABLE wordTable(theWord);")
+
+    var theWord = ""
+
+    var gameSwitch = 0
+
+    println("")

```

Comment:
All these print lines here are not looking good, try to have them in a separate file and call them from there